src/backend/api/meals.js

```diff
@@ -1,15 +1,120 @@
+const { response } = require("express");
 const express = require("express");
+const { useDebugValue } = require("react");
+const { insert, where } = require("../database");
 const router = express.Router();
 const knex = require("../database");
 
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
+    // knex syntax for selecting things.
     const titles = await knex("meals").select("title");
     response.json(titles);
+    console.log("in/api/meals");
+    const requestQuery = Object.keys(request.query);
+    const matchQuery = [];
+    requestQuery.some((item) => {
+      if (query.includes(item)) {
+        matchQuery.push(item);
+      }
+    });
```

Comment:
I don't follow what you're doing here, you're not using anywhere else the matchQuery..